release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -19,9 +19,9 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 {product-title} {product-version} clusters are available at https://console.redhat.com/openshift. From the {hybrid-console}, you can deploy {product-title} clusters to either on-premises or cloud environments.
 
 // Double check OP system versions
-{product-title} {product-version} is supported on {op-system-base-full} 8.8 and a later version of {op-system-base} 8 that is released before End of Life of {product-title} {product-version}. {product-title} {product-version} is also supported on {op-system-first} {product-version}. To understand {op-system-base} versions used by {op-system}, see link:https://access.redhat.com/articles/6907891[{op-system-base} Versions Utilized by {op-system-first} and {product-title}] (Knowledgebase article).
+{product-title} {product-version} is supported on {op-system-base-full} 9.0 and a later version of {op-system-base} 9 that is released before End of Life of {product-title} {product-version}. {product-title} {product-version} is also supported on {op-system-first} {product-version}. To understand {op-system-base} versions used by {op-system}, see link:https://access.redhat.com/articles/6907891[{op-system-base} Versions Utilized by {op-system-first} and {product-title}] (Knowledgebase article).
 
-You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines. {op-system-base} machines are deprecated in {product-title} 4.16 and will be removed in a future release.
+You must use {op-system} machines for the control plane and for the compute machines. {op-system-base} machines are deprecated in {product-title} 4.16 and will be removed in a future release.
```

Comment:
```suggestion
You must use {op-system} machines for the control plane and the compute machines. {op-system-base} machines are not available in this release.
```
@dfitzmau I will remove the last sentence for RHEL machines availability.

Reply:
I think the statement should be removed in favour of a release note.

Reply:
yeah, i think we can go ahead and just remove the statement here. there's a release note available for the removal of RHEL worker nodes in the Removed Features section 😄

Reply:
Hi @tmalove . Is still see the following statement in the preview:

![Screenshot From 2025-06-06 10-36-03](https://github.com/user-attachments/assets/a866b28e-10e7-4a80-a773-91cae75e78f9)

This statement should be removed?

Reply:
Removed RHEL sentence.